packages/cli-platform-ios/src/commands/runIOS/index.ts

```diff
@@ -610,6 +630,21 @@ export default {
       name: '--no-packager',
       description: 'Do not launch packager while building',
     },
+    {
+      name: '--port <number>',
+      default: process.env.RCT_METRO_PORT || 8081,
+      parse: Number,
+    },
+    {
+      name: '--terminal <string>',
+      description:
+        'Launches the Metro Bundler in a new window using the specified terminal path.',
+      default: getDefaultUserTerminal(),
+    },
+    {
+      name: '--no-packager',
+      description: 'Do not launch packager while building',
```

Comment:
since `--no-packager` was moved from `build` commands, it should have appropriate description (both iOS and Android)